### PR TITLE
BookManager

### PR DESCRIPTION
**Pull request by ngthanh219**
## Related Tickets:
- [#32948](https://edu-redmine.sun-asterisk.vn/issues/32948)

## WHAT this PR does?
- Bao gồm: ```Thêm, sửa, xóa, tìm kiếm, show chi tiết book```, ```Popup chọn category mới trong form thêm, sửa book```

## Notes Impacted Areas:
- N/A.

## Attachments:
library_book

### CODE REVIEW

**Review by Cho99 — APPROVED** [review]

**Comment by ngthanh219** [comment]
ready

File: resources/js/cate_popup_form.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            removeClass();
+        },
+        error: function(error) {
+            console.log(error);
+        },
+        complete: function(res) {
+            // alert(data);
+        }
+    })
+});
```
Comment (huyenttm-0901) [inline resources/js/cate_popup_form.js]: EOF

**Comment by ngthanh219** [comment]
ready

File: resources/js/cate_popup_form.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            'child_name': child_name,
+        },
+        success: function (res) {
+            var html = '<option value="' + res.dataChild.id + '">' + res.dataChild.name +
+                '</option>';
+            $('#category_id').append(html);
+            removeClass();
+        },
+        error: function (error) {
+            console.log(error);
```
Comment (huyenttm-0901) [inline resources/js/cate_popup_form.js]: bỏ console.log

**Comment by ngthanh219** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**